frontend/packages/console-dynamic-plugin-sdk/src/app/components/status/PopoverStatus.scss

```
@@ -0,0 +1,7 @@
+.popover-status {
```

```suggestion
.co-popover-status {
```
We prefix with `.co-`

---

Further, an element should not exist without a block.  This should really be `.co-popover-status-button`.